core/trino-main/src/main/java/io/trino/operator/BigintGroupByHash.java

```diff
@@ -376,8 +376,6 @@ private int registerGroupId(Block dictionary, int positionInDictionary)
     {
```

Comment:
It seems it mostly helps for:
```
BenchmarkGroupByHashOnSimulatedData.groupBy             0   BIGINT_10K_GROUPS         ADD  avgt   10  10,671 ± 0,364  ns/op
BenchmarkGroupByHashOnSimulatedData.groupBy             0   BIGINT_10K_GROUPS  GET_GROUPS  avgt   10  13,108 ± 0,266  ns/op
```
why not other cases?
Does this makes a diff in practice

Reply:
It's a slight change. I don't think we should expect huge gains here. It's rather a simplification than a performance trick.